yaksh/models.py

```diff
@@ -36,10 +36,13 @@
     )
 
 question_types = (
-        ("mcq", "Multiple Choice"),
+        ("mcq", "Single Correct Choice"),
         ("mcc", "Multiple Correct Choices"),
         ("code", "Code"),
         ("upload", "Assignment Upload"),
+        ("integer", "Answer in Integer"),
+        ("string", "Answer in String"),
+        ("float", "Answer in Float"),
```

Comment:
Instead of giving name as `Answer in Float`, Can't we write `Fill in the blanks(Float or Float value)`?. Same in above cases also.